### ISSUE TITLE
migrate crds from hard code to generate

### ISSUE THREAD

**Issue by jichenjc** [kubernetes-sigs/cluster-api-provider-openstack#193]
actually this comes from #155 , when doing test found crd folder contains something can be generated by same tool with crds folder name (see following sample).

```
root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api-provider-openstack# ls config/crds
openstackproviderconfig_v1alpha1_openstackclusterproviderspec.yaml  openstackproviderconfig_v1alpha1_openstackclusterproviderstatus.yaml
root@kvm-017751:~/go/src/sigs.k8s.io/cluster-api-provider-openstack# ls config/crd
openstackproviderconfig_v1alpha1.yaml
```

**Comment by jichenjc** [comment]
I will work on this

**Comment by chrigl** [comment]
/close

This is also closed via #189
